README.md

```diff
@@ -63,7 +63,7 @@ For more detail on taking advantage of new Bicep constructs that replace an equi
 
 **What unique benefits do you get with Bicep?**
 
-1. Day 0 resource provider support. Any ARM resource — whether in private or public preview or GA — can be provisioned using Bicep. It should be noted that several Azure resources, such as Azure Databricks, Azure DevOps, and App registrations, do not fall under the category of ARM resources and are therefore outside the scope of this discussion.
+1. Day 0 resource provider support. Any ARM resource — whether in private or public preview or GA — can be provisioned using Bicep. It should be noted that several Azure resources, such as Azure Databricks, Azure DevOps, and App registrations, do not fall under the category of Azure Resource Manager (ARM) resources and are therefore outside the scope of this discussion.
```

Comment:
I think it would be clearer if we changed the first usage of ARM, not the second. So change:

"Day 0 resource provider support. Any ARM resource ..."

to 

"Day 0 resource provider support. Any Azure Resource Manager (ARM) resource ..."

It might also be good to clarify that you an easy way to know if something is an ARM resource is if it has an ARM Resource ID.